x-pack/plugin/build.gradle

```diff
@@ -94,6 +94,18 @@ tasks.named("yamlRestTest").configure {
   dependsOn "copyExtraResources"
 }
 
+tasks.named("transformV7RestTests").configure({ task ->
+  task.replaceKeyInDo("license.delete", "xpack-license.delete")
+  task.replaceKeyInDo("license.get", "xpack-license.get")
+  task.replaceKeyInDo("license.get_basic_status", "xpack-license.get_basic_status")
+  task.replaceKeyInDo("license.get_trial_status", "xpack-license.get_trial_status")
+  task.replaceKeyInDo("license.post", "xpack-license.post")
+  task.replaceKeyInDo("license.post_start_basic", "xpack-license.post_start_basic")
+  task.replaceKeyInDo("license.post_start_trial", "xpack-license.post_start_trial")
+
+  task.addAllowedWarningRegex(".*_xpack/license.* is deprecated.*")
```

Comment:
should this be allowed or just warnings ? I think we can require it since the modified spec only has the deprecated variant.

Reply:
Ooh, good point. I'll check!

Reply:
Alright, I looked into this for a little while, and I don't think just warnings ends up working very well for this, at least as it's currently written. 

It requires a test name (see the `InjectWarnings` constructor), which works well enough on its own, but we also (sometimes) need to allow warnings on the `do` sections within `setup` or `teardown`, and that happens to end up being the case here.

That's not to say that we couldn't rework the machinery a bit to make that **possible**, but it's more than just a tweak on this PR.